nixos/doc/manual/configuration/customizing-packages.section.md

```diff
@@ -72,3 +97,6 @@ that depends on `emacs` will be passed your customised instance.
 (However, the value `pkgs.emacs` in `nixpkgs.config.packageOverrides`
 refers to the original rather than overridden instance, to prevent an
 infinite recursion.)
+
+[0]: https://search.nixos.org/packages/query=python39Packages.requests
+[1]: https://search.nixos.org/packages/query=passExtensions.pass-otp
```

Comment:
Please inline the links.

Reply:
Ok, sure! Can do!

Mind explaining why that is preferable? o.0

Reply:
(I'll do it all the same - just asking for my own ignorance's sake)

Reply:
We don't really do that somewhere else and the link is right where you use it. Also we don't need to maintain a link list at the end of the file.